### PR TITLE
Add support to flush directly to New Relic Insights Event API 

### PR DESCRIPTION
**Pull request by Kav91**
Additional details provided & updated within readme.

### CODE REVIEW

File: pkg/backends/newrelic/newrelic.go [hunk 1]
```diff
@@ -303,10 +328,23 @@ func NewClientFromViper(v *viper.Viper) (gostatsd.Backend, error) {
 	nr.SetDefault("enable-http2", defaultEnableHttp2)
 	nr.SetDefault("user-agent", defaultUserAgent)
 
+	//New Relic Config Defaults & Recommendations
+	v.SetDefault("statser-type", "null")
+	v.SetDefault("flush-interval", "10s")
+	if v.GetDuration("flush-interval").Seconds() < 10 {
+		log.Warnf("[%s] flush-interval below 10s, recommended to be >= 10s", BackendName)
+	} else if v.GetDuration("flush-interval").Seconds() == 10 {
+		log.Infof("[%s] flush-interval default 10s", BackendName)
```
Comment (tiedotguy) [inline pkg/backends/newrelic/newrelic.go]: It doesn't make a lot of sense to only log this when it's 10s, I'd just make it an "else" not an "else if".  That being said, I  suggest pushing this down to line 393.

File: pkg/backends/newrelic/newrelic.go [hunk 2]
```diff
@@ -303,10 +328,23 @@ func NewClientFromViper(v *viper.Viper) (gostatsd.Backend, error) {
...
+	}
+	if v.GetString("statser-type") == "null" {
+		log.Infof("[%s] internal metrics OFF, to enable set 'statser-type' to StatserInternal", BackendName)
```
Comment (tiedotguy) [inline pkg/backends/newrelic/newrelic.go]: The internal statser type is `internal` (actually it's the default switch case, but "internal" is the intended type :)), maybe something more like "to enable set 'statser-type' to 'logging' or 'internal'"?

Comment (Kav91) [reply pkg/backends/newrelic/newrelic.go]: oh yes, of course thats much better 👍

File: pkg/backends/newrelic/newrelic.go (hunk 1 above)
Comment (Kav91) [reply pkg/backends/newrelic/newrelic.go]: Shuffled and switched to else.

**Comment by TylerLubeck** [comment]
Hi @tiedotguy! This code is going to unblock a bunch of work for me. Any chance there's an update on the release?

**Comment by tiedotguy** [comment]
Yep, should have done this sooner, sorry!

**Pull request merged**